middlewares/accesslog/logger_test.go

```diff
@@ -92,7 +138,7 @@ func printLogdata(logdata []byte) string {
 	return fmt.Sprintf(
```

Comment:
Hmm its the error message we have to provide in a "static" way to the assert/require invocations. Therefore we have to do it at least once for each test case. I improved the message a bit and inlined the formatting into the `requireValidLogData`, so that we only do it once :)